docs/website/01_framework/controlRoutines.md

```diff
@@ -59,14 +63,14 @@ auto pft = Ikarus::StandardArcLength{};
 auto pft = Ikarus::LoadControlWithSubsidiaryFunction{};
 auto pft = Ikarus::DisplacementControl{controlledIndices};
 ```
-The default path following type is the `#!cpp Ikarus::StandardArcLength{}`.
+The default path-following type is the `#!cpp Ikarus::StandardArcLength{}`.
 
 It is to note that in the current implementation, it is assumed that the external forces are given by 
```

Comment:
Can you wrap this node in an admonition? https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types e.g. note or warning?

Reply:
done